src/connect.js

```diff
@@ -20,7 +20,16 @@
  * @return {Function}
  */
 function connect(params) {
-  // write code here
+
+  const paramsList = Object.assign({}, params)
+
+  return (paramsEstra) => {
```

Comment:
Good, but you can write simply 
```suggestion
  return (extraParams) => ({ ...params, ... extraParams });
```